### PR TITLE
[FR] Update `Community/Forum/Off-Topic`

### PR DESCRIPTION
**Pull request by Corne2Plum3**
Proposal to update the French translation of the wiki page  of `Community/Forum/Off-Topic`, which was recently updated by #6921.

* added the mention "abreviated by OT" at the beginning of the page.
* added the new parts of the page according to #6921.
* fixed the 7th refence at the bottom, where the archive's year is incorrect (2022 intead of 2017 in the original wiki page)

It's my first contribution here, so maybe formatting and grammar is incorrect...

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

**Comment by Bambinexx** [comment]
I'll review stuff tomorrow, too tired for now

File: wiki/Community/Forum/Off-Topic/fr.md [hunk 1]
```diff
@@ -1,9 +1,85 @@
...
+[**Off-Topic**](https://osu.ppy.sh/community/forums/52) (abrégé par ***OT***) est un forum destiné à permettre à la communauté d'avoir des conversations sans sujet particulier. Les fils de discussion peuvent souvent être aléatoires, changer de sujet, et/ou avoir plusieurs sujets simultanément. Les sujets peuvent aller de la vie quotidienne aux opinions sur le jeu, en passant par le jeu de rôle, les blagues et le méta-contenu, les bêtises, ou tout ce qui se trouve entre les deux.
+
+## Histoire
+
+Le forum de [General Discussions](https://osu.ppy.sh/community/forums/7) *(discussions générales)* à initialement contenu n'importe quelle sujet que les gens voulait en parler. Pendant que les jeux de forums commençaient à polluer le forum, les personnes ont suggérés faire un sous-forum spécifique pour ces "fils auto-entretenus" [[1]][r]. Le 19 Février 2009, [peppy](/wiki/People/peppy) crée ce sous-forum suggéré : *Forum Games/Random* [[2]][r] [[3]][r].
```
Comment (Biokax) [inline wiki/Community/Forum/Off-Topic/fr.md:7]:
```suggestion
Le forum de [General Discussions](https://osu.ppy.sh/community/forums/7) *(discussions générales)* à initialement servie aux gens de pouvoir discuter de n'importe quelle sujet. Étant donné que les forums de jeux ont commencé à polluer le sous-forum, les personnes ont suggérés faire un sous-forum spécifique pour ces "fils auto-entretenus" [[1]][r]. Le 19 Février 2009, [peppy](/wiki/People/peppy) crée ce sous-forum suggéré : *Forum Games/Random* [[2]][r] [[3]][r].
```

File: wiki/Community/Forum/Off-Topic/fr.md [hunk 2]
```diff
@@ -1,9 +1,85 @@
...
+
+Le nouveau sous-forum a créé une distinction qui fait que *General Discussion* était utilisé pour les discussions sérieuses, pendant que *Forum Games/Random* était utilisé pour les discussions non-sérieuses. Etant donné la nature de *Forum Games/Random*, les personnes interrompait parfois les jeux du sous-forum avec des conversations aléatoires. [[4]][r]. Le 2 Février 2010, *Forum Games/Random* a été renommé *Off-Topic* et le sous-forum *Forum Games* a été créé [[5]][r].
```
Comment (Biokax) [inline wiki/Community/Forum/Off-Topic/fr.md:9]:
```suggestion
Le nouveau sous-forum a créé une distinction qui fait que *General Discussion* était utilisé pour les discussions sérieuses, pendant que *Forum Games/Random* était utilisé pour les discussions non-sérieuses. Etant donné la nature de *Forum Games/Random*, les personnes interrompait parfois les sous-forum de jeux avec des conversations aléatoires. [[4]][r]. Le 2 Février 2010, *Forum Games/Random* a été renommé *Off-Topic* et le sous-forum *Forum Games* a été créé [[5]][r].
```

File: wiki/Community/Forum/Off-Topic/fr.md [hunk 3]
```diff
@@ -1,9 +1,85 @@
...
+
+Deux sous-forums en plus seront créés plus tard: *Mafia* le 14 Septembre 2011 (pour les jeux comme Mafia ou Loup-Garou) et *Surveys* le 5 Mars 2014 (pour les sondages).
+
+## Traditions et culture
+
+### Valhalla
+
+Etant donnée la nature de OT, la majorité des fils et discussions étaient de qualité très pauvre. Les fils étaient rarement considérés bons, mais quand ces fils atteignaient ce niveau de qualité, les utilisateurs disaient qu'ils avaient atteint Valhalla.
```
Comment (Biokax) [inline wiki/Community/Forum/Off-Topic/fr.md:17]:
```suggestion
Étant donnée la nature de OT, la majorité des fils et discussions étaient de très pauvre qualité. Les fils étaient rarement considérés bons, mais quand ces fils atteignaient ce niveau de qualité, les utilisateurs disaient qu'ils avaient atteint le Valhalla.
```

File: wiki/Community/Forum/Off-Topic/fr.md [hunk 4]
```diff
@@ -1,9 +1,85 @@
...
+
+Le 9 Juin 2017, [Ephemeral](https://osu.ppy.sh/users/102335) a affirmé une version légèrement différente de la tradition dans les [règles de Off-Topic](https://osu.ppy.sh/community/forums/topics/604424) en imposant aux personnes à poster au moins 3 filles d'animés qui dansent sur les fils d'anniversaires. Cette règle a même été ajouté dans les règles officiels d'osu! [[7]][r] jusqu'au 11 Novembre 2019, quand il a été retiré dans un [effort de nettoyage](https://github.com/ppy/osu-wiki/pull/2655), mais cette règle persiste encore dans les règles du sous-forum Off-Topic.
+
+### L'exil
+
+L'**exil** est un concept de modération non-officiel introduit par [Ephemeral](https://osu.ppy.sh/users/102335) dans [the grand edict thread](https://osu.ppy.sh/community/forums/topics/604424) en réponse contre les personnes qui spammaient le sous-forum et qui créent du contenu embêtant. Le système d'exil est appliqué en supprimant n'importe quel fil, et même parfois des posts par la personne qui est exilé. Il sert d'indicateur que la personne est sur le point d'être lourdement sanctionné par des sanctions plus officielles par la modération.
```
Comment (Biokax) [inline wiki/Community/Forum/Off-Topic/fr.md:33]:
```suggestion
L'**exil** est un concept de modération non-officiel introduit par [Ephemeral](https://osu.ppy.sh/users/102335) dans [the grand edict thread](https://osu.ppy.sh/community/forums/topics/604424) en réponse aux personnes qui spammaient le sous-forum, notamment par la création de contenu embêtant. Le système d'exil est appliqué en supprimant n'importe quel fil, et même parfois des posts de la personne qui est exilée. Il sert d'indicateur quand une personne est sur le point d'être lourdement sanctionné par des moyens plus officielles par la modération.
```

File: wiki/Community/Forum/Off-Topic/fr.md [hunk 5]
```diff
@@ -1,9 +1,85 @@
...
+
+### Denizens
+
+Depuis que le sous-forum Off-Topic est principalement composé des mêmes personnes qui discutent entre eux dans les fils, tout le monde se connaît. Ceux qui restent suffisamment longtemps pour être considérés comme utilisateur réguliers sont appelés "**denizens**" *("citoyens" en français)*. Ce terme a été utilisé pour la première fois par [Ephemeral](https://osu.ppy.sh/users/102335) sur [ce post](https://osu.ppy.sh/community/forums/posts/1607086), mais n'a pas été adopté par les utilisateurs réguliers qu'à partir de la populaire série de fils [OT!News](https://osu.ppy.sh/community/forums/topics/577518).
```
Comment (Biokax) [inline wiki/Community/Forum/Off-Topic/fr.md:37]:
```suggestion
Depuis que le sous-forum Off-Topic est principalement composé des mêmes personnes qui discutent entre elles dans les fils, tout le monde se connaît. Ceux qui restent suffisamment longtemps pour être considérés comme utilisateur réguliers sont appelés "**denizens**" *("citoyens" en français)*. Ce terme a été utilisé pour la première fois par [Ephemeral](https://osu.ppy.sh/users/102335) sur [ce post](https://osu.ppy.sh/community/forums/posts/1607086), mais n'a pas été adopté par les utilisateurs réguliers qu'à partir de la populaire série de fils [OT!News](https://osu.ppy.sh/community/forums/topics/577518).
```

**Review by Biokax — CHANGES_REQUESTED** [review]

File: wiki/Community/Forum/Off-Topic/fr.md [hunk 6]
```diff
@@ -1,9 +1,85 @@
----
-outdated: true
-outdated_since: d2b98adabd345f841cb5cdddc15b281fe79c2908
-stub: true
----
-
 # Off-Topic
 
-[**Off-Topic**](https://osu.ppy.sh/community/forums/52) est un forum destiné à permettre à la communauté d'avoir des conversations sans sujet particulier. Les fils de discussion peuvent souvent être aléatoires, changer de sujet, et/ou avoir plusieurs sujets simultanément. Les sujets peuvent aller de la vie quotidienne aux opinions sur le jeu, en passant par le jeu de rôle, les blagues et le méta-contenu, les bêtises, ou tout ce qui se trouve entre les deux.
+[**Off-Topic**](https://osu.ppy.sh/community/forums/52) (abrégé par ***OT***) est un forum destiné à permettre à la communauté d'avoir des conversations sans sujet particulier. Les fils de discussion peuvent souvent être aléatoires, changer de sujet, et/ou avoir plusieurs sujets simultanément. Les sujets peuvent aller de la vie quotidienne aux opinions sur le jeu, en passant par le jeu de rôle, les blagues et le méta-contenu, les bêtises, ou tout ce qui se trouve entre les deux.
```
[the remark this answers is not in the document]
Comment (Corne2Plum3) [reply wiki/Community/Forum/Off-Topic/fr.md:3]: I'll put this modification as is.

File: wiki/Community/Forum/Off-Topic/fr.md (hunk 2 above)
Comment (Corne2Plum3) [reply wiki/Community/Forum/Off-Topic/fr.md:9]: Seems better than what I made

File: wiki/Community/Forum/Off-Topic/fr.md (hunk 3 above)
Comment (Corne2Plum3) [reply wiki/Community/Forum/Off-Topic/fr.md:17]: ok

File: wiki/Community/Forum/Off-Topic/fr.md [hunk 7]
```diff
@@ -1,9 +1,85 @@
...
+
+L'**exil** est un concept de modération non-officiel introduit par [Ephemeral](https://osu.ppy.sh/users/102335) dans [the grand edict thread](https://osu.ppy.sh/community/forums/topics/604424) en réponse aux personnes qui spammaient le sous-forum, notamment par la création de contenu embêtant. Le système d'exil est appliqué en supprimant n'importe quel fil, et même parfois des posts de la personne qui est exilée. Il sert d'indicateur quand une personne est sur le point d'être lourdement sanctionné par des moyens plus officielles par la modération.
+
+### Denizens
+
+Depuis que le sous-forum Off-Topic est principalement composé des mêmes personnes qui discutent entre elles dans les fils, tout le monde se connaît. Ceux qui restent suffisamment longtemps pour être considérés comme utilisateur réguliers sont appelés "**denizens**" *("citoyens" en français)*. Ce terme a été utilisé pour la première fois par [Ephemeral](https://osu.ppy.sh/users/102335) sur [ce post](https://osu.ppy.sh/community/forums/posts/1607086), mais n'a pas été adopté par les utilisateurs réguliers qu'à partir de la populaire série de fils [OT!News](https://osu.ppy.sh/community/forums/topics/577518).
```
Comment (Ganondorf-1) [inline wiki/Community/Forum/Off-Topic/fr.md:37]:
```suggestion
Puisque que le sous-forum Off-Topic est principalement composé des mêmes utilisateurs qui discutent entre eux dans les fils, tout le monde se connaît. Ceux qui restent suffisamment longtemps pour être considérés comme utilisateurs réguliers sont appelés "**denizens**" *("citoyens" en français)*. Ce terme a été utilisé pour la première fois par [Ephemeral](https://osu.ppy.sh/users/102335) sur [ce post](https://osu.ppy.sh/community/forums/posts/1607086), mais n'a pas été adopté par les utilisateurs réguliers qu'à partir de la populaire série de fils [OT!News](https://osu.ppy.sh/community/forums/topics/577518).
```

File: wiki/Community/Forum/Off-Topic/fr.md [hunk 8]
```diff
@@ -1,9 +1,85 @@
...
+
+### Le lore d'Off-Topic
+
+Le **lore d'Off-Topic** est un effort lancé par [abraker](https://osu.ppy.sh/users/4635891) et entretenu par les autres *denizens* pour créer un arc narratif des différents fils que les personnes ont crées dans OT. Le meilleur de cet effort a été complié dans le fil [OT!Chronicles](https://osu.ppy.sh/community/forums/posts/6230570), qui a créé une timeline de plusieurs fils de role-play improvisés dans OT.
```
Comment (Ganondorf-1) [inline wiki/Community/Forum/Off-Topic/fr.md:41]:
```suggestion
Le **lore d'Off-Topic** est un effort lancé par [abraker](https://osu.ppy.sh/users/4635891) et entretenu par les autres *denizens* pour créer un arc narratif des différents fils que les utilisateurs ont créé dans OT. Le meilleur de cet effort a été compilé dans le fil [OT!Chronicles](https://osu.ppy.sh/community/forums/posts/6230570), qui a créé une timeline de plusieurs fils de role-play improvisés dans OT.
```

File: wiki/Community/Forum/Off-Topic/fr.md [hunk 9]
```diff
@@ -1,9 +1,85 @@
...
+
+### Déraillement de fil
+
+Le **déraillement de fil** est une occurrence commune dans le sous-forum Off-Topic à cause de sa nature "hors-sujet". Cela signifie que les fils crées avec un sujet en particulier ne peuvent pas forcément garder ce même sujet.
```
Comment (Ganondorf-1) [inline wiki/Community/Forum/Off-Topic/fr.md:45]:
```suggestion
Le **déraillement de fil** est une occurrence commune dans le sous-forum Off-Topic à cause de sa nature "hors-sujet". Cela signifie que les fils créés avec un sujet en particulier peuvent ne pas forcément garder ce même sujet.
```

File: wiki/Community/Forum/Off-Topic/fr.md [hunk 10]
```diff
@@ -1,9 +1,85 @@
...
+
+## Fils notables
+
+Bien que la majorité des fils dans OT sont des bêtises ou d'autres contenus sans importance, il y a quelques fils notables que les personnes non familiers avec le sous-forum pourraient trouver intéressant.
```
Comment (Ganondorf-1) [inline wiki/Community/Forum/Off-Topic/fr.md:49]:
```suggestion
Bien que la majorité des fils dans OT sont des bêtises ou d'autres contenus sans importance, il y a quelques fils notables que les utilisateurs non familiers avec le sous-forum pourraient trouver intéressant.
```

File: wiki/Community/Forum/Off-Topic/fr.md [hunk 11]
```diff
@@ -1,9 +1,85 @@
...
+
+### La série OT!News
+
+Cette série de fils est un très bon moyen de montrer les hauts et les bas de OT sans voir la plupart des bêtises.
+
+<https://osu.ppy.sh/community/forums/posts/5942617>
+
+### OT COURT II: A-Breaking Laboratory & R.O.S.Q. vs Asht. Co
+
+OT court est un élément du lore d'OT qui imite de manière détaillé un procès. Cette organisation a été improvisé lors de sa création, et contre toute attente, est resté du début jusqu'à la fin sans perdre d'intérêt ou sans être déraillé.
```
Comment (Ganondorf-1) [inline wiki/Community/Forum/Off-Topic/fr.md:59]:
```suggestion
OT court est un élément du lore d'OT qui imite de manière détaillée un procès. Son organisation a été improvisée lors de sa création, et contre toute attente, est restée du début jusqu'à la fin sans perdre d'intérêt ou sans être déraillée.
```

File: wiki/Community/Forum/Off-Topic/fr.md [hunk 12]
```diff
@@ -1,9 +1,85 @@
...
+
+<https://osu.ppy.sh/community/forums/posts/6956061>
+
+### Tad the Tadpole
+
+Ce fil est une blague qui a été développé dans le lore d'OT. C'est un bon exemple de comment les *denizens* peuvent créer des fils-blagues à propos de chacun, et par la suite créer une narration à partir des interactions de ce fil. Il montre également la nature imprévisible de l'évolution des fils et comment des bêtises et des choses inattendus peuvent apparaître à partir de ces fils.
```
Comment (Ganondorf-1) [inline wiki/Community/Forum/Off-Topic/fr.md:65]:
```suggestion
Ce fil est une blague qui a été développée dans le lore d'OT. C'est un bon exemple de comment les *denizens* peuvent créer des fils-blagues à propos de chacun, et par la suite créer une narration à partir des interactions de ce fil. Il montre également la nature imprévisible de l'évolution des fils et comment des bêtises et des choses inattendues peuvent apparaître à partir de ces fils.
```

File: wiki/Community/Forum/Off-Topic/fr.md [hunk 13]
```diff
@@ -1,9 +1,85 @@
...
+
+<https://osu.ppy.sh/community/forums/posts/7126163>
+
+### Testing
+
+C'est l'un des fils de role-playing résultant du lore d'OT, qui a crée un  improvisé récit intéressant impliquant un méga-télétubbie divin qui fait des ravages dans OT et que les *denizens* essaient de vaincre.
```
Comment (Ganondorf-1) [inline wiki/Community/Forum/Off-Topic/fr.md:71]:
```suggestion
C'est l'un des fils de role-playing résultant du lore d'OT, qui a créé un intéressant récit improvisé impliquant un méga-télétubby divin qui fait des ravages dans OT et que les *denizens* essaient de vaincre.
```

File: wiki/Community/Forum/Off-Topic/fr.md [hunk 14]
```diff
@@ -1,9 +1,85 @@
...
+
+<https://osu.ppy.sh/community/forums/posts/7549082>
+
+## References
+
+1. [Discussion à propos de la création d'un sous-forum pour les jeux de forum](https://osu.ppy.sh/community/forums/posts/80316)
+2. [L'annoncement de peppy du nouveau sous-forum](https://osu.ppy.sh/community/forums/posts/8814)
```
Comment (Ganondorf-1) [inline wiki/Community/Forum/Off-Topic/fr.md:78]:
```suggestion
2. [L'annonce de peppy du nouveau sous-forum](https://osu.ppy.sh/community/forums/posts/8814)
```

File: wiki/Community/Forum/Off-Topic/fr.md [hunk 15]
```diff
@@ -1,9 +1,85 @@
...
+3. [Mention de l'ancien nom du sous-forum Off-Topic](https://osu.ppy.sh/community/forums/posts/132900)
+4. [Mention de comment les jeux de forums ont été trollés](https://osu.ppy.sh/community/forums/posts/313614)
+5. [Réactions de personnes du changement du sous-forum Of-Topic](https://osu.ppy.sh/community/forums/posts/316732)
```
Comment (Ganondorf-1) [inline wiki/Community/Forum/Off-Topic/fr.md:81]:
```suggestion
5. [Réactions d'utilisateurs au changement du sous-forum Off-Topic](https://osu.ppy.sh/community/forums/posts/316732)
```

File: wiki/Community/Forum/Off-Topic/fr.md [hunk 16]
```diff
@@ -1,9 +1,85 @@
...
+6. [Les recherches d'abraker à propos de la tradition des filles d'animés qui dansent sur les fils d'anniversaires](https://osu.ppy.sh/community/forums/topics/1525770)
+7. [Archive du 17 août 2017 des règles officiels d'osu!](http://web.archive.org/web/20170817161329/http://osu.ppy.sh/help/wiki/Rules)
```
Comment (Ganondorf-1) [inline wiki/Community/Forum/Off-Topic/fr.md:83]:
```suggestion
7. [Archive du 17 août 2017 des règles officielles d'osu!](http://web.archive.org/web/20170817161329/http://osu.ppy.sh/help/wiki/Rules)
```

File: wiki/Community/Forum/Off-Topic/fr.md [hunk 17]
```diff
@@ -1,9 +1,85 @@
----
-outdated: true
-outdated_since: d2b98adabd345f841cb5cdddc15b281fe79c2908
-stub: true
----
-
 # Off-Topic
 
-[**Off-Topic**](https://osu.ppy.sh/community/forums/52) est un forum destiné à permettre à la communauté d'avoir des conversations sans sujet particulier. Les fils de discussion peuvent souvent être aléatoires, changer de sujet, et/ou avoir plusieurs sujets simultanément. Les sujets peuvent aller de la vie quotidienne aux opinions sur le jeu, en passant par le jeu de rôle, les blagues et le méta-contenu, les bêtises, ou tout ce qui se trouve entre les deux.
+[**Off-Topic**](https://osu.ppy.sh/community/forums/52) (abrégé par ***OT***) est un forum qui a pour but de à permettre à la communauté d'avoir des conversations sans sujet particulier. Les fils de discussion peuvent souvent être aléatoires, changer de sujet, et/ou avoir plusieurs sujets simultanément. Les sujets peuvent aller de la vie quotidienne aux opinions sur le jeu, en passant par le jeu de rôle, les blagues et le méta-contenu, les bêtises, ou tout ce qui se trouve entre les deux.
```
Comment (Ganondorf-1) [inline wiki/Community/Forum/Off-Topic/fr.md:3]:
```suggestion
[**Off-Topic**](https://osu.ppy.sh/community/forums/52) (abrégé par ***OT***) est un forum qui a pour but de permettre à la communauté d'avoir des conversations sans sujet particulier. Les fils de discussion peuvent souvent être aléatoires, changer de sujet, et/ou avoir plusieurs sujets simultanément. Les sujets peuvent aller de la vie quotidienne aux opinions sur le jeu, en passant par le jeu de rôle, les blagues et le méta-contenu, les bêtises, ou tout ce qui se trouve entre les deux.
```

**Review by Ganondorf-1 — CHANGES_REQUESTED** [review]

File: wiki/Community/Forum/Off-Topic/fr.md [hunk 18]
```diff
@@ -1,9 +1,85 @@
...
+
+Deux sous-forums en plus seront créés plus tard: *Mafia* le 14 Septembre 2011 (pour les jeux comme Mafia ou Loup-Garou) et *Surveys* le 5 Mars 2014 (pour les sondages).
+
+## Traditions et culture
+
+### Valhalla
+
+Étant donnée la nature de OT, la majorité des fils et discussions étaient de très pauvre qualité. Les fils étaient rarement considérés bons, mais quand ces fils atteignaient ce niveau de qualité, les utilisateurs disaient qu'ils avaient atteint le Valhalla.
+
+Le terme a été utilisé pour la première fois par [Ephemeral](https://osu.ppy.sh/users/102335) dans le fil [ITT: We're Dirty](https://osu.ppy.sh/community/forums/topics/40298), et plusieurs fois par la suite. Depuis, les autres personnes du sous-forum ont adopté ce terme.
```
[the remark this answers is not in the document]
Comment (Corne2Plum3) [reply wiki/Community/Forum/Off-Topic/fr.md:19]: Je ne suis pas sûr à propos de cette suggestion parce que dans la version originale le mot '_people_' est utilisé et non '_users'_.

[the remark this answers is not in the document]
Comment (Corne2Plum3) [reply wiki/Community/Forum/Off-Topic/fr.md]: Bon en fait tu suggère d'utiliser le mot "utilisateurs" au lieu de "personnes" dans beaucoup de suggestions. Par souci de cohérence, je vais quand même appliquer cette suggestion.

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**